lib/datadog/core/configuration/components.rb

```diff
@@ -89,6 +98,7 @@ def build_crashtracker(settings, agent_settings, logger:)
           :telemetry,
           :tracer,
           :crashtracker,
+          :errortracker,
```

Comment:
```suggestion
          :error_tracker,
```

Reply:
addressed in ecfdce57e4880c87d13c07923e31ef91a92f64e7